Mage/Task/BuiltIn/Symfony2/SymfonyAbstractTask.php

```diff
@@ -26,7 +29,13 @@ protected function getAppPath()
         } else {
             $defaultAppPath = 'app/console';
         }
-
-        return $this->getConfig()->general('symfony_app_path', $defaultAppPath);
+        
+        $path = '';
+        $niceness = intval($this->getParameter('niceness', 0));
```

Comment:
By the way, such thing shouldn't be done in `getAppPath` method, because you expect to get absolute or relative path to Symfony's `command` script. In the other hand there is no method for calling explicitly concrete Symfony2 command .